blocks/columns/columns.css

```diff
@@ -168,6 +168,12 @@
   margin-left: 0;
 }
 
+.columns.left-section-half-width > div > div > div {
```

Comment:
I see this is used for the first block on the page, instead of adding this and using a columns block, could `hero horizontal tabs` be used?
here's an example https://main--sunstar-foundation--hlxsites.hlx.page/award

Reply:
yes, I can use that. But confusing for the author, given there are no tabs.

I feel following is better for authors to follow:
![image](https://github.com/hlxsites/sunstar-foundation/assets/119420493/76ac63b1-ad35-4d5a-bce3-a61d546453d4)

instead of below:

![image](https://github.com/hlxsites/sunstar-foundation/assets/119420493/213d60d7-2268-40c5-add1-e40a32858386)

but I am fine changing it, if we want to avoid this new variant.

Reply:
In the spirit of less is more, I think we can reuse. Arguably `Hero Horizontal Tabs` is a misnomer and should be rename to just `Hero` at some point.

Reply:
sure

Reply:
Done. I have removed the variant